app/src/main/java/fr/publicissapient/planningpoker/model/Card.kt

```diff
@@ -0,0 +1,9 @@
+package fr.publicissapient.planningpoker.model
+
+import androidx.compose.ui.text.AnnotatedString
+
+data class Card(
+	val imageResourceId: Int,
+	val description: AnnotatedString,
+	val value: Int
```

Comment:
le nom de ce variable pourrait être plus "explicite" j'avoue je ne comprends pas tout de suite ce que c'est la `valeur` d'une carte ?

Reply:
Yes me too, what do you suggest?